clicache/integration-test2/Cluster.cs

```diff
@@ -32,7 +32,6 @@ public class Cluster : IDisposable
         private int serverCount_;
         private bool started_;
         private List<Locator> locators_;
```

Comment:
Why remove the servers set? They are going to be needed to stop individual servers.

Reply:
Was totally just fixing complaints, didn't actually look at the substance 😞 

Apparently we are never actually using the list the to stop individual servers, that should probably be the real fix instead of just deleting the list.

For record keeping, LGTM was pointing out that we added to the list of servers, but never actually retrieved any information from that list later.